src/Models/UsersAssociatedApps.php

```diff
@@ -43,4 +45,51 @@ public function initialize()
 
         $this->setSource('users_associated_apps');
     }
+
+    /**
+     * Upload Files.
+     *
+     * @todo move this to the baka class
+     *
+     * @return void
+     */
+    public function afterSave()
+    {
+        if (!$this->validateIsActive()) {
+            $parentUser = Di::getDefault()->getUserData();
+            $userInactiveConfirmation = new UserInactiveConfirmation($parentUser);
+            $userInactiveConfirmation->setFrom($parentUser);
+
+            $this->getUser()->notify($userInactiveConfirmation);
+        }
+    }
+
+    /**
+     * Checks whether or not a user is active on the current app's company
+     *
+     * @return bool
+     */
+    public function validateIsActive() : bool
+    {
+        return $this->user_active ? true : false;
+    }
+
+    /**
+     * Get record by user's id
+     *
+     * @param int $userId
+     *
+     * @return UsersAssociatedApps
+     */
+    public static function getByUserId(int $userId): self
+    {
+        return self::findFirstOrFail([
+            "conditions" => "apps_id = :apps_id: and users_id = :users_id: and companies_id = :companies_id: and is_deleted = 0",
+            "bind" => [
+                "apps_id" => 1,
```

Comment:
@rwhite27 why is this hardcoded?

Reply:
lol forgot to change it back XD @kaioken